arch/arm64/boot/dts/qcom/msm8953-asus-ze553kl.dts

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2024, Antony Kurniawan S. <postmarketos@smankusors.com>
```

Comment:
?

Reply:
this should be your real name and your email (and not mine 😭), in case we want to bring this to upstream @nationpwned